src/data_provider/tests/sysInfoRpmPackageManager/RpmPackageManager_test.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wazuh SysInfo
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * March 16, 2021.
```

Comment:
Hi Team

This file is new, the date must be this month or last month.

Regards
Hanes